hazelcast/src/main/java/com/hazelcast/config/ConfigXmlGenerator.java

```diff
@@ -619,21 +619,6 @@ private void mapStoreConfigXmlGenerator(StringBuilder xml, MapConfig m) {
         }
     }
 
-    private void nearCacheConfigXmlGenerator(StringBuilder xml, NearCacheConfig n) {
```

Comment:
I think that we may be actually missing this method's invocation to output the near-cache configuration in `ConfigXmlGenerator.generate`.